pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -94,7 +94,7 @@ self: super: {
 
     # the tests for shell-conduit on Darwin illegitimatey assume non-GNU echo
     # see: https://github.com/psibi/shell-conduit/issues/12
-    doCheck = !pkgs.stdenv.hostPlatform.isDarwin;
+    doCheck = !pkgs.stdenv.isDarwin;
```

Comment:
I believe that the `hostPlatform` bit is actually superior and we should probably change all occurrences that don't use it to this style. I think it matters during cross-compilation, but I am not entirely sure.